### PR TITLE
Survey entity

### PR DESCRIPTION
**Pull request by GagandeepLubana**
Completed the Survey entity class. Originally was working accidentally on UserDataInterface, fixed that.

### CODE REVIEW

**Review by realdevvora — CHANGES_REQUESTED** [review]
Just remove the local.properties file and it should be good.

Also make sure to add the getters/setters onto the CRC document so we can reference them if we need to.

**Review by realdevvora — DISMISSED** [review]

File: app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/interfaces/survey_class_imports_implementations/UserDataInterface.java [hunk 1]
```diff
@@ -1,4 +1,6 @@
 package com.example.coursematchdaddy.clean_architecture_layers.entities.interfaces.survey_class_imports_implementations;
 
+import java.util.ArrayList;
+
 public interface UserDataInterface {
```
Comment (uyoyo0) [inline app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/interfaces/survey_class_imports_implementations/UserDataInterface.java:5]: In the design doc this interface has a method _**`public HashMap<String, Object> getUserSurveyData()`**_
Add that method and implement it in any classes that implement this interface

**Review by uyoyo0 — CHANGES_REQUESTED** [review]
See my comment in the `UserDataInterface` file

**Review by JaspreetKhela — CHANGES_REQUESTED** [review]
Please add comments to your code. Everything else looks good.

File: app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/interfaces/survey_class_imports_implementations/UserDataInterface.java (hunk 1 above)
Comment (GagandeepLubana) [reply app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/interfaces/survey_class_imports_implementations/UserDataInterface.java:5]: That wasn't supposed to be in this pull req, its for another issue, I changed it by accident.

**Review by JaspreetKhela — APPROVED** [review]

**Review by uyoyo0 — APPROVED** [review]
Looks good

**Review by realdevvora — APPROVED** [review]

**Comment by realdevvora** [comment]
I think the way to fix the merge conflict is to remove the local.properties file, since we should all technically have our own version of it, and it isnt suppoed to be pushed to main anyways

**Review by realdevvora — APPROVED** [review]

**Review by uyoyo0 — APPROVED** [review]
Looks good

**Comment by M4N4VS1NGH** [comment]
Looks fine

**Review by JaspreetKhela — APPROVED** [review]

**Pull request merged**